tools/codegen/api/lazy.py

```diff
@@ -8,22 +8,22 @@
                                      scalarT, scalarTypeT)
 
 valueT = BaseCppType('torch::lazy', 'Value')
-
+# this is a bad hack. I need to refactor the data model to represent each arg in the schema as an object,
+# making it easier to represent special properties of an arg.
+tensorListValueT = BaseCppType('torch::lazy', 'Value')
 
 def process_ir_type(typ: Type) -> Union[BaseCType, VectorCType, OptionalCType, ListCType]:
     """
     This function takes a type from NativeFunctions and converts it for use with
-    lazy tensor codegen.  Currently its output is used in several places, and so far
-    it has been possible for them to all use the same conversions, but that may not be
-    optimal or possible in the finished system.
+    lazy tensor codegen.
 
     Type conversion for lazy currently consists of
-     (1) changing Tensor-like things into Value-like things
+     (1) changing at::Tensors into lazy::Values
```

Comment:
lmao, these conversions rules sound so much more precise now.

Reply:
i hope you're not joking hah. i was trying!